include/opt-sched/Scheduler/OptSchedTarget.h

```diff
@@ -8,10 +8,11 @@
 #ifndef LLVM_OPT_SCHED_TARGET_H
 #define LLVM_OPT_SCHED_TARGET_H
 
-#include "opt-sched/Scheduler/OptSchedDDGWrapperBase.h"
-#include "opt-sched/Scheduler/data_dep.h"
-#include "opt-sched/Scheduler/defines.h"
-#include "opt-sched/Scheduler/machine_model.h"
+#include "OptSched/include/opt-sched/Scheduler/OptSchedDDGWrapperBase.h"
+#include "OptSched/include/opt-sched/Scheduler/config.h"
+#include "OptSched/include/opt-sched/Scheduler/data_dep.h"
+#include "OptSched/include/opt-sched/Scheduler/defines.h"
+#include "OptSched/include/opt-sched/Scheduler/machine_model.h"
```

Comment:
```suggestion
#include "opt-sched/Scheduler/OptSchedDDGWrapperBase.h"
#include "opt-sched/Scheduler/config.h"
#include "opt-sched/Scheduler/data_dep.h"
#include "opt-sched/Scheduler/defines.h"
#include "opt-sched/Scheduler/machine_model.h"
```

The `OptSched/include` is not part of the include path. Don't write it like this, even if it works. If it didn't work without it, you should fix the build system include paths.

Reply:
This was done due to the way we are registering OptSched with the static build -- I will try again with the original include paths, but I remember having issues.

Reply:
If I remember correctly, I resolved this in my branch of LLVM/OptSched. I think it just involved setting the proper include directories the AMDGPU or CodeGen CMakeLists.

Reply:
Alright -- yeah I think things will be more clear once we have our LLVM fork up to date. If we want to include from within AMDGPU then I can change these paths -- I'll need to double check the registration concern though.

Reply:
Not sure why I originally changed the include paths -- I was able to get it to work by including relevant files in AMDGPU/CMakeLists.txt